### DISCUSSION TITLE
Using PyOMP in Tardis?

### DISCUSSION THREAD

**Discussion by DrTodd13** [tardis-sn/tardis#2199] [General]
One of the Numba core developers suggested that Tardis might be interested in using PyOMP which is a Numba derivative that supports OpenMP directly in Python. I'm also a Numba core dev and the initial creator of PyOMP.  Is this a good spot to start to have that discussion?

**Comment by wkerzendorf** [comment]
Thanks for bringing this up. Very excited to talk about this - what do you have in mind?

**Comment by DrTodd13** [comment]
We have a PyOMP working group Thursdays at 9am Pacific time.  Would you be able to attend that?  If that time works for you and other tardis people then get me their emails and I can extend the meeting invitation.

**Comment by Rodot-** [comment]
I can think of a few places that this would be very useful for streamlining some of the code, I would be interested in trying it out

**Comment by DrTodd13** [comment]
github.com/Python-for-HPC/PyOMP is the main page for PyOMP.  However, since PyOMP is a full distribution of Numba with OpenMP additions, the real main repo is a Numba fork in github.com/Python-for-HPC/numbaWithOpenmp.

Numba provides a ton of versions that support many combinations of Python, NumPy, operating system, architecture, etc.  We don't have the extensive build infrastructure in place yet to do that at the moment and thus right now we only provide pre-packaged PyOMP for Linux x86-64 with glibc >= 2.29 for one Python and NumPy version combination.  You can get that distribution with the following command.  Building manually is possible for other environments but painful.

conda install drtodd13::numba cffi -c conda-forge –override-channels

What does everyone think?  Do you want to come to one of our regular PyOMP working group meetings or do you want to have a separate one off meeting?
